cob_script_server/src/simple_script_server/simple_script_server.py

```diff
@@ -634,29 +644,54 @@ def compose_trajectory(self, component_name, parameter_name):
 		return (traj_msg, 0)
 
 	def calculate_point_time(self, start_pos, end_pos, default_vel, default_acc):
+		if isinstance(default_vel, float):
+			default_vel = numpy.array([default_vel for _ in start_pos])
+
+		if isinstance(default_acc, float):
+			default_acc = numpy.array([default_acc for _ in start_pos])
+
 		try:
-			d_max = max(list(abs(numpy.array(start_pos) - numpy.array(end_pos))))
-			t1 = default_vel / default_acc
-			s1 = default_acc / 2.0 * t1**2
-			if (2 * s1 < d_max):
-				# with constant velocity phase (acc, const vel, dec)
-				# 1st phase: accelerate from v=0 to v=default_vel with a=default_acc in t=t1
-				# 2nd phase: constante velocity with v=default_vel and t=t2
-				# 3rd phase: decceleration (analog to 1st phase)
-				s2 = d_max - 2 * s1
-				t2 = s2 / default_vel
-				t = 2 * t1 + t2
-			else:
-				# without constant velocity phase (only acc and dec)
-				# 1st phase: accelerate from v=0 to v=default_vel with a=default_acc in t=t1
-				# 2nd phase: missing because distance is to short (we already reached the distance with the acc and dec phase)
-				# 3rd phase: decceleration (analog to 1st phase)
-				t = math.sqrt(d_max / default_acc)
-			point_time = max(t, 0.4)	# use minimal point_time
+			# d_max: Distance traveled in the move by each joint
+			d_max = numpy.abs(numpy.array(start_pos) - numpy.array(end_pos))
```

Comment:
this is an array of distances now, not a scalar distance_max anymore...`d_max` name should be adjusted

Reply:
The same goes for all of them actually, numpy makes it look the same of course, but `default_vel`, `default_acc`, `d_max`, `t1`, `s1`, `t` as all arrays. I think the comment is clear about it and I prefer to not make the name more complicated but keep the expressions short.

Reply:
I rather meant that `d_max` is not even `max` anymore...it's just an array of joint distances...the array part is ok...